packages/react/src/side-nav/docs/overview.mdx

```diff
@@ -86,4 +86,113 @@ The background of the `SideNav` must match the background it is being placed on.
 <DontHeading />
 
 - create more than 3 levels of nesting
-- place on the right-hand side of content.
+- place on the right-hand side of content
+- keep all nested lists open when there are many items.
```

Comment:
What does this mean exactly? I'm confused

Reply:
Essentially don't use `alwaysOpen` on long lists of nav items, that's what `openOnNav` is for. Open to new wording.

Reply:
I see. Perhaps we need a Do of use the "openOnNav" as the primary method? And Don't use "alwaysOpen" when there are many navigation items with many sub items?

Reply:
Yeah, that's a more complete approach, I like that.

How about this:

```md
DO:
- use `openOnNav` as a primary variant.

DON'T:
- use `alwaysOpen` when you have a long list of navigation items.
```

Reply:
primary variant doesn't seem to mean much here? I know I used primary method above, and method does make it a big clearer, but still not clear enough imo. Perhaps it should be something along the lines of "use `openOnNav` unless you have very good reason not to do so"?

Also, I think we need guidance on when we do the click into a nav with sub options and only show those options like the current export service website. Is that still a thing?

Reply:
- [ ] Get designers' input on wording